content/sensu-go/5.21/reference/agent.md

```diff
@@ -774,6 +774,8 @@ Flags:
       --namespace string                      agent namespace (default "default")
       --password string                       agent password (default "P@ssw0rd!")
       --redact string                         comma-delimited customized list of fields to redact
+      --require-fips
+      --require-openssl
```

Comment:
```suggestion
      --require-fips                              indicates whether fips support should be required in openssl
      --require-openssl                       indicates whether openssl should be required instead of go's built-in crypto
```